app/portainer/views/settings/authentication/settingsAuthenticationController.js

```diff
@@ -11,7 +11,8 @@ function($q, $scope, $state, Notifications, SettingsService, FileUploadService,
   };
 
   $scope.formValues = {
```

Comment:
Create the formValues with the entire expected object layout (add details on `LDAPSettings` expected fields).